src/util/dates.test.js

```diff
@@ -0,0 +1,9 @@
+import { formatDate } from "./dates";
```

Comment:
Note that native date formatting for JS has 2 minor differences from what you were doing with Moment:
1. AM/PM is capitalized
2. There's no leading 0 on the hour.

Tests convey the behavior.

Reply:
Updated breaking tests